data/json/recipes/armor/jewelry.json

```diff
@@ -121,5 +121,25 @@
     "qualities": [ { "id": "DRILL", "level": 1 }, { "id": "FILE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
     "//3": "Metal sawing for removing the base, rim, and primer, and filing for smoothing down the resulting edges.",
     "components": [ [ [ "wire", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "necklace_teeth",
+    "variant": "necklace_teeth_monster",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "skills_required": [ [ "fabrication", 1 ] ],
+    "time": "15 m",
+    "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_gem_setting" } ],
+    "//": "Pliers to cut and manipulate the wire.",
+    "tools": [ [ [ "pliers", -1 ] ] ],
+    "using": [ [ "cordage_short", 1 ] ],
+    "//2": "The filing quality is only there to blunt the 'impossibly-sharp' monster teeth, probably not needed for other teeth.",
+    "qualities": [ { "id": "FILE", "level": 1 } ],
+    "components": [ [ [ "wire", 1 ] ], [ [ "nm_teeth", 1 ] ] ]
```

Comment:
Wouldn’t you want something to thread the wire through the tooth, like in the case of the bullet jewellery? I guess you could just retain it by wrapping the wire tightly, but that seems like a rather unstable mounting point.

Reply:
This is 100% a valid concern but it depends on the shape of the tooth, which isn't described, if it is "triangular" with a point and a large base and a curve that can hold a wire it should hold, not problem.

If the tooth is more human like, blocky and curved outward, then indeed there is a need for a hole.

I am assuming the former because that's the tooth shape a lot of predators evolve while the latter is something generally reserved for herbivores, and I am also assuming nether monsters are somewhat predator-like on average.

Does this makes sense to you?

Reply:
I'm resolving the conversation, feel free to ping me if you still think it's wrong.